wallet/node/Main.hs

```diff
@@ -49,10 +51,18 @@ actionWithWallet sscParams nodeParams wArgs@WalletArgs {..} =
   where
     convPlugins = (, mempty) . map (\act -> ActionSpec $ \__vI __sA -> act)
     plugins :: HasConfigurations => ([WorkerSpec WalletWebMode], OutSpecs)
-    plugins = convPlugins (pluginsGT wArgs) <> walletProd wArgs
+    plugins = mconcat [ convPlugins (pluginsGT wArgs)
+                      , walletProd wArgs
+                      , acidCleanupWorker wArgs ]
+
+acidCleanupWorker :: HasConfigurations => WalletArgs -> ([WorkerSpec WalletWebMode], OutSpecs)
+acidCleanupWorker WalletArgs{..} =
+    first one $ worker mempty $ const $
+    modifyLoggerName (const "acidcleanup") $
+    cleanupAcidStatePeriodically walletAcidInterval
```

Comment:
Ok, so this will do the first checkpoint immediately on startup, and will do another after the interval of time. Good.